wallet/service/openapi.json

```diff
@@ -47,6 +47,30 @@
           }
         }
       }
+    },
+    "/v2/methods": {
+      "get": {
+        "summary": "List the available methods on the JSON-RPC API.",
+        "description": "The JSON-RPC API use behind this HTTP API can have an arbitrary setup in terms of available methods.\n\nPrior using the requests API, an application might want to check if the JSON-RPC it requires are presents. It also useful to avoid to explore the API in a die-and-repeat style.",
```

Comment:
```suggestion
        "description": "The JSON-RPC API use behind this HTTP API can have an arbitrary setup in terms of available methods.\n\nPrior using the requests API, an application might want to check if the JSON-RPC it requires are present. It is also useful to avoid exploring the API in a die-and-repeat style.",
```